userspace/plugin/plugin_types.h

```diff
@@ -126,6 +132,9 @@ typedef struct ss_plugin_extract_field
     {
 		const char** str;
 		uint64_t* u64;
+		uint32_t* u32;
+		bool* boolean;
+		const_sized_buffer** buf;
```

Comment:
What's the rationale behind the double pointer? I think this could just be `ss_plugin_byte_buffer* buf`, so that we don't force implementers to do an extra level of indirection/allocation.

Reply:
Whops! I'll fix it! 😅